props/blaster.h

```diff
@@ -260,9 +233,65 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
     }
   }
 
+   RefPtr<BufferedWavPlayer> auto_player_;
+
+  uint32_t auto_time_;
+  virtual void Fire() {
+    CheckEmpty();
+    if (empty_) return;
+#ifdef ENABLE_MOTION
+#ifdef BLASTER_JAM_PERCENTAGE
+    // If we're already jammed then we don't need to recheck. If we're not jammed then check if we just jammed.
+    is_jammed_ = is_jammed_ ? true : CheckJam(BLASTER_JAM_PERCENTAGE);
+    if (is_jammed_) {
+      SaberBase::DoEffect(EFFECT_JAM, 0);
+      return;
+    }
+#endif
+#endif
+    if (blaster_mode == MODE_AUTO) {
+      // Set up the autofire pairing if the font suits it.
+      SelectAutoFirePair(); 
+      SaberBase::SetLockup(LOCKUP_AUTOFIRE);
+      SaberBase::DoBeginLockup();
+      auto_firing_ = true;
+      auto_time_ = millis();
+
+      // GET auto.wav here
+      if (!auto_player_) {
+        auto_player_ = GetWavPlayerPlaying(&SFX_auto);
```

Comment:
If there happens to be more than one SFX_auto running, this might end up finding the wrong one. Maybe we should find the one that was started most recently?

Reply:
ok

Reply:
Having a tough time with filtering only players playing auto.wav. Do we get &SFX_auto during bgnlock playing, or do we need to wait to check until after bgnlock is done?
I keep coming up with an empty hook when fishing for a matching player.

Reply:
commit named filter help needed

Reply:
Is this really needed? When would there be more than one SFX_auto running that would confise GetWavPlayerPlaying(&SFX_auto) ?

Reply:
I think this part can somehow re-use the code now used for KILL_OLD_PLAYERS.
I'll try to wrap my head around this maybe this week.